client/src/main/scala/eventstore/package.scala

```diff
@@ -0,0 +1,37 @@
+
+import eventstore.{akka => a}
+import java.{util => ju}
+
+// TODO(AHJ): Remove this package object after 7.1.0
```

Comment:
If it was my call, I would not have this package object at all. I included it in order to make it a bit easier for clients to upgrade. But as this PR builds on a PR that has breaking changes wrt. Joda, we can just as well just break this and put release notes that say specify package changes noted below, as well as the Joda removal.